joomla/joomla-cms
# [5.1] Add Tag Relationship in ContentSerializer.php

The work was already done by Joomla Team and other contributors what I did is just to use the TagApiSerializerTrait in ContentSerializer. It should be added everywhere where the tags are supported but to make it simple I added it just to ContentSerializer. Hope it helps. Tested it locally on my setup with Postman and it works.

Pull Request for Issue # .
No issues for this pull request

### Summary of Changes
Add the Tag Relationship for ContentSerializer


### Testing Instructions
1. Apply the patch
2. Run this command   curl -X GET -H "X-Joomla-Token: YOUR_JOOMLA_API_TOKEN" -H "Accept: application/vnd.api+json" --url "https://example.org/api/index.php/v1/content/articles"
3. It should return a response in JSON:API format containing the tags relationship

### Actual result BEFORE applying this Pull Request
The tag relationship does not appear when doing a GET request on /api/index.php/v1/content/articles 


### Expected result AFTER applying this Pull Request
The tag relationship appears successfully alongside the other relationships like category and created_by


### Link to documentations
Please select:
